tests/inputFields.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { test, expect } from '@playwright/test';
+
+
+test.describe('Input fields', () => {  
+test.beforeEach( async({page}) => {
+  await page.goto('/')
+  await page.getByRole('link', { name: 'Pet Types' }).click();
+  await page.getByRole('heading', { name: 'Pet Types' }).click()
+})
+
+test('Update pet type', async ({page}) => {
+const tableRow = page.locator('tbody tr')
+await tableRow.getByRole('button', {name: 'Edit'}).first().click()
+await expect(page.getByRole('heading')).toContainText('Edit Pet Type');
```

Comment:
If you want to use exact assertion, use the `toHaveText()` method. `toContainText()` need to be used when you want to make a partial validation of the text